src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The excavator started working");

```

Comment:
The message in the `doWork` method should be more informative and indicate that it is an `Excavator` type of `Machine` that is working. Consider changing the message to something like: 'Excavator started its work.'